Content.Shared/Wieldable/WieldableSystem.cs

```diff
@@ -276,9 +288,7 @@ public bool TryUnwield(EntityUid used, WieldableComponent component, EntityUid u
         if (ev.Cancelled)
             return false;
 
-        component.Wielded = false;
         var targEv = new ItemUnwieldedEvent(user);
-
         RaiseLocalEvent(used, targEv);
         return true;
     }
```

Comment:
... Is this subscribing to its own event? Might as well just roll it into it and save the performance.

Reply:
sure thing